.git-hooks/pre-commit

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require 'json'
+
+# Prevent a Ruby stack trace from appearing when we interrupt the hook.
+Signal.trap('INT') do
+  puts 'Hook run interrupted'
+  exit 130
+end
+
+puts "======================================================="
+puts "Running Verification for tokens, secret keys, etc..."
+puts "======================================================="
+puts ""
+
+# Declare Postman var Files that need to be audited and include the the key fields
+# that need to be verified to be blank
+ENV_FILES = {
+  'postman/Data_at_the_Point_of_Care_Sandbox.postman_environment.json' => [
+    'macaroon',
+    'PRIVATE_KEY',
+    'key_id',
+    'jwt_token',
+    'access_token',
+    'job-id',
+    'file_name',
+    'patient_npi'
+  ],
+  'postman/DPC_API.postman_globals.json' => [
+    'organization_id',
+    'attribution_group'
+  ]
+}.freeze
+
+# Only need to check the files if they have been committed with changes
+FILES_MODIFIED = `git diff --cached --name-only`.chomp.split("\n")
+
+errors = []
+
+FILES_MODIFIED.each do |file|
+  if ENV_FILES.key?(file)
+    file_data = JSON.parse(File.read(file))
+    file_data['values'].each do |v|
+      if ENV_FILES[file].include?(v['key']) && !v['value'].empty?
```

Comment:
Can we add a little more logic here?  Arguable, if the JSON content of these files is not parsable, the error message is useful to the dev:

```
 > git commit -m "invalid JSON"

=======================================================
Running Verification for tokens, secret keys, etc...
=======================================================

Traceback (most recent call last):
	4: from .git-hooks/pre-commit:41:in `<main>'
	3: from .git-hooks/pre-commit:41:in `each'
	2: from .git-hooks/pre-commit:43:in `block in <main>'
	1: from /Users/davidgreene/.rvm/rubies/ruby-2.6.3/lib/ruby/2.6.0/json/common.rb:156:in `parse'
/Users/davidgreene/.rvm/rubies/ruby-2.6.3/lib/ruby/2.6.0/json/common.rb:156:in `parse': 767: unexpected token at 'organization_id: asdf (JSON::ParserError)
'
```

But if the file is missing entirely, I'd prefer not to have my commit blocked:

```
> git commit -m "missing file"
=======================================================
Running Verification for tokens, secret keys, etc...
=======================================================

Traceback (most recent call last):
	3: from .git-hooks/pre-commit:41:in `<main>'
	2: from .git-hooks/pre-commit:41:in `each'
	1: from .git-hooks/pre-commit:43:in `block in <main>'
.git-hooks/pre-commit:43:in `read': No such file or directory @ rb_sysopen - postman/DPC_API.postman_globals.json (Errno::ENOENT)
```

Reply:
@dhgreene These change requests have been added in! Check the screenshots above to see the results.